geometry/spatial-operations/README.md

```diff
@@ -2,20 +2,20 @@
 
 Find the union, difference, or intersection of two geometries.
 
-![](SpatialOperations.png)
+![Image of spatial operations](SpatialOperations.png)
+
+## Use case
+
+Determining the spatial relationships between points, lines or polygons is a fundamental concept in GIS. For example, a data analyst may need to find the intersection between areas with a high concentration of medical emergency situations and the service area of a hospital.
 
 ## How to use the sample
 
 The sample provides a drop down on the top, where you can select a geometry operation. When you choose a geometry operation, the application performs this operation between the overlapping polygons and applies the result to the geometries.
```

Comment:
looks like there's an unneeded space at the start of this sentence

Reply:
I can't see it! Maybe it's the github diff acting up?

Reply:
I think it's the diff yes! When it shows a plus for new change it auto bumps the text over and makes it look like there's a space I've learned. Sorry!